hypha/apply/activity/admin.py

```diff
@@ -8,7 +8,7 @@ class MessageInline(admin.TabularInline):
     readonly_fields = ('type', 'recipient', 'content', 'status', 'external_id')
     can_delete = False
 
-    def has_add_permission(self, request):
+    def has_add_permission(self, request, obj):
```

Comment:
Should we make the same update in the other call to "has_add_permission"? To "has_delete_permission"?

Reply:
I don't think we need to change it for other calls, because I don't see anything in the release notes regarding this and I checked the code there is no change in BaseModelAdmin, they change it only for InlineModelAdmin(especially for `has_add_permission`). So, I think we are good.